validator/main.go

```diff
@@ -121,6 +122,10 @@ var (
 	metricsPort                   int
 	defaultGPUWorkloadConfigFlag  string
 	disableDevCharSymlinkCreation bool
+	driverManagedByOperator       bool
+	hostRootFlag                  string
+	driverRootFlag                string
+	devRootFlag                   string
```

Comment:
Instead of separate flags how about the following:
```
type options struct {

	type hostPaths struct {
		root string
		driverRoot string
		devRoot string
	}
}
```
(or even a named struct for the `hostPaths`).

If we don't want to update the functions signatures to pass options instead, then we can create a global variable:
```
var hostPaths hostPathsType{}
```
and refer to this when setting the command line arguments.